content/docs/guides/vercel.md

```diff
@@ -142,6 +142,10 @@ The integration supports automatic deletion of obsolete preview branches when th
 4. In the **Vercel integration** drawer, select the **Branches** tab.
 5. Check **Automatically delete obsolete Neon branches**.
 
+<Admonition type="note">
+Avoid creating child branches on automatically created preview branches. This will prevent the preview branch from being automatically deleted. In Neon, child branches must be deleted before you can delete a parent branch. 
```

Comment:
```suggestion
Avoid creating child branches on automatically created preview branches, as it will prevent a preview branch from being automatically deleted. In Neon, child branches must be deleted before you can delete a parent branch. 
```